### PR TITLE
add link to functions integration

### PR DESCRIPTION
**Pull request by danmoseley**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/reference/aspire-faq.yml](https://github.com/dotnet/docs-aspire/blob/b9da826c738d88619b2b4e8a69ae9f71b42dacd2/docs/reference/aspire-faq.yml) | [Frequently asked questions about .NET Aspire](https://review.learn.microsoft.com/en-us/dotnet/aspire/reference/aspire-faq?branch=pr-en-us-2082) |

### CODE REVIEW

**Comment by danmoseley** [comment]
BTW @IEvangelist is https://learn.microsoft.com/en-us/dotnet/aspire/reference/aspire-faq#are-azure-functions-supported-in--net-aspire- in standard format? Maybe it's just me but the H1/body/H1/body type formatting is hard to read/ a bit ugly compared to our topics that have longer content blocks.

**Review by captainsafia — APPROVED** [review]
🚀

File: docs/reference/aspire-faq.yml [hunk 1]
```diff
@@ -97,8 +97,7 @@ sections:
       - question: |
           Are Azure Functions supported in .NET Aspire?
         answer: |
-          We have no specific support for Azure Functions thus far in .NET Aspire, however it's a target
-          execution environment for deployment that we are planning to support in future.
+          Aspire now has [preview support for integrating Azure Functions into your app](https://learn.microsoft.com/en-us/dotnet/aspire/serverless/functions).
```
Comment (IEvangelist) [inline docs/reference/aspire-faq.yml:100]:
```suggestion
          .NET Aspire has [preview support for integrating Azure Functions into your app](../serverless/functions.md).
```

**Review by IEvangelist — APPROVED** [review]

**Comment by IEvangelist** [comment]
> BTW @IEvangelist is https://learn.microsoft.com/en-us/dotnet/aspire/reference/aspire-faq#are-azure-functions-supported-in--net-aspire- in standard format? Maybe it's just me but the H1/body/H1/body type formatting is hard to read/ a bit ugly compared to our topics that have longer content blocks.

Yeah, this is the standard FAQ template. We could avoid the template that's provided and just write custom Markdown.

**Pull request merged**